frontend/src/design/components/layout/DefaultSidebar.js

```diff
@@ -28,7 +28,7 @@ export const DefaultSidebar = ({ openDrawer, onOpenDrawerChange }) => {
       title: 'Catalog',
       path: '/console/catalog',
       icon: <VscBook size={15} />,
-      active: config.modules.datasets.active || config.modules.dashboards.active
+      active: isModuleEnabled(ModuleNames.CATALOG)
     };
 
     const datasetsSection = {
```

Comment:
Out of curiosity - why do we not pass isModuleEnabled(...) for Datasets, Shares, and Glossaries?

Reply:
We still need to decide on the behavior for module enablement for `Datasets`, because it would potentially disable other modules as you mention like `Shares`, `Glossaries`,etc. But as for this PR, we're just doing the refactoring (potential bug fixes also)